src/services/Api.tsx

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import { config } from '../config';
+
+export function newRequest(method: string, values: any, queryParams: any, data: any) {
```

Comment:
Why not use ES6 arrow syntax? `function` is so old school ;)
Plus with typescript, the arrow function fixes the `this` issues too -- bonus.